### PR TITLE
Restore missing frontend assets

### PR DESCRIPTION
**Pull request by hueyy**
## What does this PR do?
* Restores missing frontend assets (#968)
* Fixes the paths for images, etc.
* Sets `output.publicPath` in Webpack based on environment variables

## ✅ Pull Request checklist

- [x] Is this code complete?
- [x] Are tests done/modified?
- [x] Are docs written for this PR? (if applicable)

## 🚨 Is this a breaking change for API clients?
No

## :squirrel: Deploy notes
* Need to re-upload static assets to S3

### CODE REVIEW

**Comment by hueyy** [comment]
Wait, I just realised we might not use `.env` files in non-dev environments. But this is okay because we build frontend assets on our own machines which have `.env` files. And even if the environment variables are loaded w/o a `.env` file, the import of the `.env` file will silently fail and the loaded environment variables are still used. Okay, seems fine.

File: frontend/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "ajv": "^6.7.0",
     "aphrodite": "^2.2.3",
+    "dotenv": "^6.2.0",
```
Comment (ChristopherHammond13) [inline frontend/package.json]: Would this actually be a dev dependency as opposed to a general one?

File: frontend/webpack.dev.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
+require('dotenv')
```
Comment (ChristopherHammond13) [inline frontend/webpack.dev.js]: Do we actually need this logic in `webpack.dev.js`? I'm not sure there's a situation where we'd want to build dev assets for prod, unless I've missed something of course.

File: frontend/package.json (hunk 1 above)
Comment (hueyy) [reply frontend/package.json]: Fair point.

File: frontend/webpack.dev.js (hunk 2 above)
Comment (hueyy) [reply frontend/webpack.dev.js]: That makes a lot of sense.

**Pull request merged**